css/styles.css

```diff
@@ -1,19 +1,38 @@
-
-body {
-    height: 100vh;
-    width: 100vw;
```

Comment:
i noticed that there was an unexpected "x overflow" sideways scroll bar. in my experience, this can often because by the use of `100vw` in your CSS for an element's `width` value. not always, but, in this case, i removed all the instances of `100vw` i could find until i found that `.content` was the offending class.

i removed it, added some more flex box magic, and now you shouldn't have any issues with horizontal scrolling going forward!

![fixed-x-scroll](https://user-images.githubusercontent.com/4302376/223624235-ac34330f-2142-468c-9246-c83377378ef3.png)